pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -68,6 +68,86 @@ const (
+	// for the readiness probe
+	DefaultControllerReadinessProbeFailureThreshold = 3
+
+	// DefaultReadinessProbeSuccessThreshold is the default probe success threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeSuccessThreshold = 3
+
+	// DefaultReadinessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbeTimeoutSeconds = 60
+
+	// DefaultLivenessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbeInitialDelaySeconds = 60
+
+	// DefaultLivenessProbePeriodSeconds is the default probe period (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbePeriodSeconds = 15
+
+	// DefaultLivenessProbeFailureThreshold is the default probe failure threshold
+	// for the liveness probe
+	DefaultControllerLivenessProbeFailureThreshold = 4
+
+	// DefaultLivenessProbeSuccessThreshold is the default probe success threshold
+	// for the liveness probe
+	DefaultControllerLivenessProbeSuccessThreshold = 1
+
+	// DefaultLivenessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbeTimeoutSeconds = 5
+
+	// DefaultReadinessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeInitialDelaySeconds = 10
+
+	// DefaultReadinessProbePeriodSeconds is the default probe period (in seconds)
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbePeriodSeconds = 10
+
+	// DefaultReadinessProbeFailureThreshold is the default probe failure threshold
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeFailureThreshold = 30
+
+	// DefaultReadinessProbeSuccessThreshold is the default probe success threshold
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeSuccessThreshold = 1
+
+	// DefaultReadinessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the readiness probe
+	DefaultSegmentStoreReadinessProbeTimeoutSeconds = 5
+
+	// DefaultLivenessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the liveness probe
+	DefaultSegmentStoreLivenessProbeInitialDelaySeconds = 300
+
+	// DefaultLivenessProbePeriodSeconds is the default probe period (in seconds)
+	// for the liveness probe
+	DefaultSegmentStoreLivenessProbePeriodSeconds = 15
+
+	// DefaultLivenessProbeFailureThreshold is the default probe failure threshold
+	// for the liveness probe
+	DefaultSegmentStoreLivenessProbeFailureThreshold = 4
+
+	// DefaultLivenessProbeSuccessThreshold is the default probe success threshold
+	// for the liveness probe
+	DefaultSegmentStoreLivenessProbeSuccessThreshold = 1
+
+	// DefaultLivenessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the liveness probe
+	DefaultSegmentStoreLivenessProbeTimeoutSeconds = 5
```

Comment:
Again, need a timeout of 60 (=15*4).

Reply:
@jkhalack `timeoutseconds` is the time taken to complete one readiness check/liveness check. More details can be found here.(https://stackoverflow.com/questions/63308717/what-is-the-role-of-timeoutseconds-in-kubernetes-liveness-readiness-probes) . Since liveness is not taking much time kept the timeout as `5` seconds

Reply:
Thank you for the explanation, @anishakj 
The official doc is confusing 😊 
Will approve the PR.